azure-pipelines.yml

```
@@ -45,26 +45,254 @@ variables:
-    skipInstall: true
-    unitTestRunsettings: '$(System.DefaultWorkingDirectory)\helper.runsettings'
-
-# step from template @ nf-tools repo
-# report error
-- template: azure-pipelines-templates/discord-webhook-task.yml@templates  
-  parameters:
-    status: 'failure'
-    webhookUrl: '$(DiscordWebhook)'
-    message: ''
+stages:
+- stage: BuildPrep
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)